src/main/resources/userData/popularWorkouts.json

```diff
@@ -0,0 +1 @@
"Triceps","Shoulders"],"equipment":["Machine"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Pronated Cable Forearm Curls","description":"Face a low cable pulley with an overhand grip on the bar. Curl the weight up by curling the wrist and not bending the elbow, then lower slowly.","bodyParts":["Forearms"],"equipment":["Cable"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Barbell Hip Thrusts","description":"Sit on the ground with your upper back against a bench and a barbell over your hips. Drive your hips upward, squeezing the glutes at the top, then lower back down slowly.","bodyParts":["Glutes"],"equipment":["Barbell"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Smith Machine Romanian Deadlifts","description":"Stand with Smith machine bar in front of thighs. Keeping knees soft, lower bar down by hinging at hips and keeping back straight, then return to standing. Repeat.","bodyParts":["Hamstrings","Glutes"],"equipment":["SmithMachine"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Weighted Lower Back Curls","description":"Lie on a hyperextension bench with weight held across the chest. Pull the chest up, and lower it slowly.","bodyParts":["LowerBack"],"equipment":["Bodyweight"],"difficulty":"Intermediate","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Machine Leg Extensions","description":"Sit in a leg extension machine with your feet secured under the pads. Extend your legs to full extension, then lower them slowly back to the starting position.","bodyParts":["Quadriceps"],"equipment":["Machine"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Dumbbell Rear Delt Flys","description":"Bend over with dumbbells, arms hanging. Raise arms out to the sides, squeezing rear delts, then lower back down.","bodyParts":["Shoulders"],"equipment":["Dumbbell"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Smith Machine JM Press","description":"Lie on a bench under Smith machine bar. Lower bar to chin level by bending elbows, then press up. Repeat.","bodyParts":["Triceps"],"equipment":["SmithMachine"],"difficulty":"Intermediate","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Dumbbell Kelso Shrugs","description":"Lie face down on an incline bench with dumbbells hanging at your sides. Shrug shoulders up, then lower back with control.","bodyParts":["UpperBack"],"equipment":["Dumbbell"],"difficulty":"Intermediate","imagePath":"","timePerRep":3000,"type":null},"sets":2,"repsPerSet":5}],"name":"Full Body Workout (90 min)","description":null,"isSaved":true}]
```

Comment:
Typo again. You changed it to "Leg Workkout" lol

Reply:
You should probably also remove the time in the title since the time is part of the description now